src/Wellcome.Dds/Wellcome.Dds.AssetDomainRepositories/Workflow/WorkflowCallRepository.cs

```diff
@@ -38,6 +39,13 @@ public async Task<WorkflowJob> CreateExpeditedWorkflowJob(string id, int? workfl
             return workflowJob;
         }
 
+        public async Task<WorkflowJob> CreateWorkflowJob(WorkflowMessage message, bool expedite = false)
+        {
+            logger.LogInformation("Creating workflow job from message: " + message);
```

Comment:
`logger.LogInformation("Creating workflow job from message: {Message}", message);` for structured logging in Kibana.

Also - is it worth dropping to Debug?

Reply:
I must get in the habit of structured logging.
I think the workflow job creation is worth keeping at INFO as something you'd want to track back in the logs. I suspect there are other things that should drop to debug though.